src/scss/layout.scss

```
@@ -60,6 +60,14 @@
   margin-bottom: $default-margin * 2;
 }
 
+.index-article {
+  background: white;
+  border-radius: 5px;
+  padding: 2em 2em 2em 2em;
```

You can use shorthand

```suggestion
  padding: 2em;
```